src/mapkit.cs

```diff
@@ -46,6 +46,10 @@
 
 namespace MapKit {
 
+#if !IOS
+	interface UIScene {}
```

Comment:
Minor: doing it this way is fair but it looks this file is using the `using` pattern (above) to achieve the same result. Might have been best to add to that (;